src/main/java/com/commercetools/sync/inventories/InventorySync.java

```diff
@@ -257,30 +298,31 @@ public InventorySync(@Nonnull final InventorySyncOptions syncOptions) {
                     return null;
                 });
         }
-        return CompletableFuture.completedFuture(null);
+        return completedFuture(null);
     }
 
     /**
      * Given an inventory entry {@code draft}, issues a request to the CTP project to create a corresponding Inventory
      * Entry.
      *
      * <p>The {@code statistics} instance is updated accordingly to whether the CTP request was carried
-     * out successfully or not. If an exception was thrown on executing the request to CTP, the optional error callback
-     * specified in the {@code syncOptions} is called.
+     * out successfully or not. If an exception was thrown on executing the request to CTP, the error handling method
+     * is called.
      *
      * @param draft the inventory entry draft to create the inventory entry from.
      * @return a future which contains an empty result after execution of the create.
      */
-    private CompletionStage<Void> attemptCreate(@Nonnull final InventoryEntryDraft draft) {
+    @SuppressFBWarnings("NP_NONNULL_PARAM_VIOLATION") // https://github.com/findbugsproject/findbugs/issues/79
+    private CompletionStage<Void> create(@Nonnull final InventoryEntryDraft draft) {
         return inventoryService.createInventoryEntry(draft)
-                               .thenAccept(createdEntry -> statistics.incrementCreated())
-                               .exceptionally(exception -> {
-                                   final Reference<Channel> supplyChannel = draft.getSupplyChannel();
-                                   final String errorMessage = format(CTP_INVENTORY_ENTRY_CREATE_FAILED, draft.getSku(),
-                                       supplyChannel != null ? supplyChannel.getId() : null);
-                                   handleError(errorMessage, exception, 1);
-                                   return null;
-                               });
+            .thenAccept(createdInventory -> statistics.incrementCreated())
```

Comment:
I honestly don't know how we can enforce a checkstyle rule for enforcing method chaining indentation correctness. But by looking at, the dots of chaining should have the same indentation level.